backend/simulators/movement_coordinator.py

```
@@ -0,0 +1,270 @@
+        # Pick next zone
+        candidates = [z for z in _ZONE_IDS if z != current_zone]
+        next_zone  = random.choice(candidates) if candidates else current_zone
+        travel_secs = _travel_seconds(current_zone, next_zone)
+
+        # Mark as traveling so simulators suppress events during transit
+        try:
+            traveling_pos = dict(position)
+            traveling_pos["traveling"] = True
+            await redis.set(ENTITY_KEY.format(eid), json.dumps(traveling_pos), ex=DATA_TTL)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)